app/Kconfig

```diff
@@ -163,6 +175,7 @@ menuconfig ZMK_SPLIT_BLE
 	depends on ZMK_BLE
 	default y
 	select BT_USER_PHY_UPDATE
```

Comment:
Can we remove this line then if we're auto updating the PHY?

Reply:
Yes we can, but then we won't be able to access the PHY value on a connection struct, so one of our logs would break without removing that. I used that log to confirm that PHY updates were still working.

Reply:
Ah, then scratch that.